src/site/_includes/partials/header-default.njk

```diff
@@ -53,7 +53,8 @@
         {{ 'i18n.header.about' | i18n(locale) }}
       </a>
     </div>
-    <web-search></web-search>
+    <web-search resultsId="search-main-results"></web-search>
```

Comment:
HTML usually doesn't camel case attributes so maybe we should rename this. I think if you do it dash cased, `results-id`, Lit is smart enough to convert that to camel case for you. Or you could use a different attribute like `for=""` or `target=""`.

Reply:
Lit didn't work its magic, so I just use the attribute directly (it does not need to be a property).